src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -354,7 +354,17 @@ private static Process StartNewVisualStudioProcess(string installationPath, int
             IntegrationHelper.KillProcess("VsJITDebugger");
             IntegrationHelper.KillProcess("dexplore");
 
-            var process = Process.Start(vsExeFile, VsLaunchArgs);
+            var processStartInfo = new ProcessStartInfo(vsExeFile, VsLaunchArgs)
+            {
+                Environment =
+                {
+                    // Clear variables set by CI builds
```

Comment:
Comment is saying what, I would focus on "why". Essentially the goal of arcade is to isolate our unit test experience to the exact SDK that we specify in global.json. In the case of VS integration tests though that is the exact opposite of what we want. We want VS to use the SDK that comes with VS, not what arcade thinks is the correct SDK.

Reply:
Updated. This what you were after?